cypress/integration/visual-testing/homepage-test.spec.js

```diff
@@ -0,0 +1,41 @@
+import { baseURL, eyesOpen } from '../../utils';
+describe('Visual  Regression Tests', () => {
+  beforeEach(() => {
+    cy.visit(baseURL);
+    eyesOpen('Home Page');
+  });
+  afterEach(() => {
+    cy.eyesClose();
+  });
+
+  it('should display sign up options', () => {
+    cy.get('#navigation-butt').click();
+    cy.get('#navigation-butt').should('be.visible');
+  });
+
+  it('should redirect to code of conduct', () => {
+    const elem = cy.get('[data-cy-href=codeofconduct]');
```

Comment:
It is generally not recommended to set variables values to a cypress command. Consider using an alias instead or chain the commands. https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html

Also, there is no need to use `return`.

```
cy.get('[data-cy-href=codeofconduct]')
	.contains('Code of Conduct')
	.invoke('attr', 'href').then(href => {
      cy.visit(`${baseURL}${href}`);
      cy.wait(2000);
      cy.get('h1')
        .should('be.visible')
        .contains('Code of Conduct');
    });
```